src/AppBundle/Form/MembershipChooseNearbyCommitteeType.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\CommitteeMembership;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+
+class MembershipChooseNearbyCommitteeType extends AbstractType
+{
+    private $choices;
+    private $memberships;
+    private $slugs;
+
+    /**
+     * @param EntityManager $em
+     */
+    public function __construct(EntityManager $em)
+    {
+        $committees = $em->getRepository(Committee::class)->findBy([], ['id' => 'desc'], 3);
```

Comment:
At this time, it's just for testing purpose. Have been replaced by a query builder in the repository in the next issue.